corpus/expressions.txt

```diff
@@ -216,6 +231,7 @@ def matchTest(x: Int): String = x match {
   case 3 => {
     "3"
   }
+  case ((i, _)) => i
```

Comment:
how come you're nesting the tuple pattern? `case (x, y) => z` is acceptable, you don't need `case ((x, y)) => z`

Reply:
fair point, I was just testing that particular use-case, since it's also an acceptable syntax, and I had to make some modifications in order for it to work.